### PR TITLE
Dashboard empty state

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: tests/acceptance/test_dashboard.py [hunk 1]
```diff
@@ -74,3 +74,10 @@ def test_new_dashboard(self):
             self.browser.get(self.path)
             self.browser.wait_until_not('.loading-indicator')
             self.browser.snapshot('new dashboard')
+
+    def test_new_dashboard_empty(self):
+        with self.feature('organizations:dashboard'):
```
Comment (billyvg) [inline tests/acceptance/test_dashboard.py]: This snapshot isn't working, you may need to just create a new user/org

Comment (lynnagara) [reply tests/acceptance/test_dashboard.py]: 👍

File: tests/acceptance/test_dashboard.py [hunk 2]
```diff
@@ -74,3 +74,28 @@ def test_new_dashboard(self):
...
+
+class EmptyDashboardTest(AcceptanceTestCase):
+    def setUp(self):
+        super(EmptyDashboardTest, self).setUp()
+        self.user = self.create_user('foo@example.com')
+        self.org = self.create_organization(
+            name='Rowdy Tiger',
+            owner=None,
+        )
+        self.team = self.create_team(organization=self.org, name='Mariachi Band')
```
Comment (billyvg) [inline tests/acceptance/test_dashboard.py]: Where is the project `bengal` coming from?

**Review by billyvg — APPROVED** [review]

File: tests/acceptance/test_dashboard.py (hunk 2 above)
Comment (lynnagara) [reply tests/acceptance/test_dashboard.py]: Where do you see `bengal`? This test case shouldn't have any project

Comment (lynnagara) [reply tests/acceptance/test_dashboard.py]: Oh, that project should exist in the other test case in this file though

Comment (billyvg) [reply tests/acceptance/test_dashboard.py]: Ah oops

**Pull request merged**